### ISSUE TITLE
Handle input with no moral charges

### ISSUE THREAD

**Issue by yang3kc** [ZeningDuan/vMFD#1]
If the input is empty or doesn't have moral charges, the code will break.

How should we fix this? Return 0 for all the metrics?

**Comment by ZeningDuan** [comment]
Yes, for empty or moral neutral inputs, return 0. This is how eMFD handles same inputs.

**Comment by yang3kc** [comment]
Thanks! I'll fix it the next time we update the package.
